src/fragment-components/tiles/a-radio-tile-group.tsx

```diff
@@ -132,7 +133,11 @@ export const ARadioTileGroup = ({
 			selected={selected}
 			{...rest}>
 				<fieldset
-				className={`bx--tile-group ${componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')}`}
+				className={cx(
+					componentObj.cssClasses?.map((cc: any) => cc.id).join(' '),
+					css`${styleObjectToString(componentObj.style)}`,
+					'bx--tile-group'
+				)}
```

Comment:
```suggestion
				className={cx(
					'bx--tile-group',
					componentObj.cssClasses?.map((cc: any) => cc.id).join(' '),
					css`${styleObjectToString(componentObj.style)}`
				)}
```

We want to overwrite bx--tile-group if required, so it needs to go at the beginning